proto/babylon/incentive/rewards.proto

```diff
@@ -0,0 +1,76 @@
+syntax = "proto3";
+package babylon.incentive;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos_proto/cosmos.proto";
+
+option go_package = "github.com/babylonlabs-io/babylon/x/incentive/types";
+
+// FinalityProviderHistoricalRewards represents the cumulative rewards ratio of the
+// finality provider per sat in that period.
+// The period is ommited here and should be part of the key used to store this structure.
+// Key: Prefix + Finality provider bech32 address + Period.
+message FinalityProviderHistoricalRewards {
+    // The cumulative rewards of that finality provider at some specific period
```

Comment:
Does this sentence mean `cumulative_rewards_per_sat` = cumulative rewards during the given period / amount of Sats delegated to this FP?

Reply:
well it is the previous FinalityProviderHistoricalRewards.CumulativeRewardsPerSat + the rewards of the given period 

That was the reason I called it cumulative, in the cosmos-sdk they had the equivalent structure that was storing the amount of rewards per share of voting power and they called [CumulativeRewardRatio](https://github.com/cosmos/cosmos-sdk/blob/d9c53bfefc1e75a3c6b09065ea8b3a836cda0d18/x/distribution/types/distribution.pb.go#L97)